tests/nnstreamer_filter_lua/unittest_filter_lua.cc

```diff
@@ -891,6 +892,87 @@ end
   sp->close (&prop, &data);
 }
 
+/**
+ * @brief Negative case with lua script with invalid data type
+ */
+TEST (nnstreamerFilterLua, dataType00_n)
+{
+  int ret;
+  void *data = NULL;
+  GstTensorFilterProperties prop;
+
+  const GstTensorFilterFramework *sp = nnstreamer_filter_find ("lua");
+  EXPECT_NE (sp, nullptr);
+
+  /**
+   * Invalid data type `uint128`. Type should be one of
+   * { float32, float64, int64, uint64, int32, uint32, int16, uint16, int8, uint8 }
+   */
+  const char *invalid_data_type = "uint128";
+  const gchar *model = g_strdup_printf ("\
```

Comment:
check returned type (char to const char)
g_strdup_printf() returns newly allocated string, need free function.
I recommend to add macro to get model string,
```
#define make_lua_model(dtype) "utTensorsInfo={num=1,dim={{1,2,2,1},},type={" dtype ",}} ...
....

in dataType00_n()
const gchar model[] = make_lua_model ("uint128");

in dataType01()
const gchar model[] = make_lua_model (gst_tensor_get_type_string (ttype));
```

Reply:
Instead of macro, I updated the code to call `g_free` for allocated strings. Thanks, PTAL!